src/Workspaces/Core/Portable/Rename/HeavyweightRenameLocations1.cs

```diff
@@ -8,9 +8,9 @@
 
 namespace Microsoft.CodeAnalysis.Rename
 {
-    internal sealed partial class RenameLocations
+    internal sealed partial class HeavyweightRenameLocations
```

Comment:
>HeavyweightRenameLocations [](http://example.com/codeflow?start=34&length=26)

Would `SymbolicRenameLocations` or `SemanticRenameLocations`  be better name?

Reply:
yup.  i like that.

Reply:
done in followup: https://github.com/dotnet/roslyn/pull/62803